backend/judging-algorithm/data/controllers/rooms-controller.ts

```diff
@@ -35,5 +41,11 @@ import * as roomsDao from "../dao/rooms-dao.js";
     return deleted;
   };
 
+  const deleteAllRooms = async (_req: any, res: any) => {
```

Comment:
is there a reason sometimes the req parameter has underscore prepended?

Reply:
It gives us an error saying "'req' is declared but its value is never read" whenever we get rid of the underscore. However, this only happens for the deleteAllRooms function